src/QuadVelocityController.cpp

```diff
@@ -122,19 +128,46 @@ bool QuadVelocityController::getVelocities(geometry_msgs::Vector3& return_veloci
                 velocities_valid = false;
             }
 
-            // Get the transforms with the stamps for readability
+            // Get the transforms without the stamps for readability
             geometry_msgs::Transform& transform = transformStamped.transform;
             geometry_msgs::Transform& oldTransform = last_transform_stamped_.transform;
 
+            // Get the yaw (z axis) rotation from the quanternion
+            double ysqr = transform.rotation.y * transform.rotation.y;
```

Comment:
I disagree in this case. Doing it via ROS requires constructing a Quaternion object, then constructing a Matrix3x3 object with the Quaternion, and then calling a method on the matrix to get the ypr. Which in the end means we constructed two objects to calculate three angles when we only needed to do a little math to get one angle.